### PR TITLE
Feature/pace oc2ioda

### LINKED ISSUE #922: Add a converter for PACE OC data
## Description
I add a converter to convert PACE ocean color data to the IODA format. As, PACE is not lunched yet, we use the simulated obs from here "https://oceandata.sci.gsfc.nasa.gov/directaccess/PACE/OCI/L2/"

### PR DESCRIPTION
**Pull request by HamidehGMAO**
## Description
This is a converter to convert PACE ocean color data to the IODA format. As, PACE is not lunched yet, we use the simulated obs from here "https://oceandata.sci.gsfc.nasa.gov/directaccess/PACE/OCI/L2/". This converter is just for the Ocean color data, I'm going to submit a separate PR for the PACE radiances data. 

### Issue(s) addressed

Link the issues to be closed with this PR
- fixes #922

### CODE REVIEW

**Review by srherbener — COMMENTED** [review]
Thanks for adding this, and for putting in a new test.

**Review by kbhargava — APPROVED** [review]
Except the one comment that @srherbener had, looks good to me.

File: src/marine/pace_oc2ioda.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    for v in ('platform', 'instrument', 'processing_level'):
+        GlobalAttrs[v] = ncd.getncattr(v)
+
+    # get QC flags, and calculate a mask from the non-missing values
+    # since L2 OC files are quite empty, need a mask applied immediately
+    # to avoid using too much memory)
+    gpd = ncd.groups['geophysical_data']
+    data_in = {}
+    data_in['l2_flags'] = gpd.variables['l2_flags'][:].ravel()
+    mask = data_in['l2_flags'] < 1073741824
```
[the remark this answers is not in the document]
Comment (HamidehGMAO) [reply src/marine/pace_oc2ioda.py:78]: Thanks @srherbener for your review. I have added a few lines clarifying that. Please let me know if you have any questions. Thanks

[the remark this answers is not in the document]
Comment (srherbener) [reply src/marine/pace_oc2ioda.py:78]: @HamidehGMAO, the new comments are very helpful and exactly what I was looking for. Thanks!

**Review by srherbener — APPROVED** [review]

**Comment by srherbener** [comment]
There are a couple coding norms errors. Once those are fixed, I'll merge this PR. Thanks!

**Review by kbhargava — APPROVED** [review]
Thanks for addressing the obs error stuff with the comment.

**Pull request merged**